make-dist

```diff
@@ -20,6 +20,27 @@ if ! git submodule sync || ! git submodule update $force --init --recursive; the
     exit 1
 fi
 
+download_boost() {
+    boost_version=$1
+    boost_md5=$2
+    boost_version_underscore=$(echo $boost_version | sed 's/\./_/g')
+    boost_fname=boost_${boost_version_underscore}.tar.bz2
+    echo "downloading boost_$boost_version..."
+    wget --no-verbose -O $boost_fname \
+         https://downloads.sourceforge.net/project/boost/boost/$boost_version/$boost_fname
```

Comment:
Should we stash a copy of this on download.ceph.com or something?

Reply:
@alfredodeza do we use a cache enabled http proxy in our CI infrastructure?

Reply:
we don't have any kind of caching in our CI